install.sh

```diff
@@ -17,11 +35,11 @@ if [[ $response = 1 ]]; then
 
   if [[ ! $response == 0 ]]; then
     currentDate=`date +%s`
-    dest="~/Library/Developer/Xcode/Templates-Backup-${currentDate}"
+    dest="${HOME}/Library/Developer/Xcode/Templates-Backup-${currentDate}"
     mkdir -p $dest
-    
+
     if cp -R ~/Library/Developer/Xcode/Templates $dest; then
-      success "Successfully backup your old templates at '$dest'"
+      success "Backup your old templates at '$dest'"
```

Comment:
Just curious why do you remove the word `Successfully`? This message is for success action right?